manifests/4.4/elasticsearch-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -52,6 +52,26 @@ metadata:
                     }
                   ]
                 }
+            },
+            {
+              "apiVersion": "logging.openshift.io/v1",
+              "kind": "Kibana",
+              "metadata": {
+                "name": "kibana"
+              },
+              "spec": {
+                "managementState": "Managed",
+                "replicas": 1,
+                "nodeSelector": {},
+                "resources": {
+                  "limits": {
+                    "memory": "1Gi"
```

Comment:
Please modify this to match our default.  This seems really large for OOTB.  Same with the requests.  Maybe alternatively, consider removing them since we provide defaults if undefined